LauraThompson/Lecture4Exercise.py

```diff
@@ -0,0 +1,257 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun May 31 11:57:51 2020
+
+@author: lthompson8
+"""
+
+#%%
+#Exercise 4.1: Write a Python program that prints all the numbers from 0 to 6 except 3 and 6.
+#Use if, loop, and continue statements
+#expected output: 0 1 2 4 5
+
+for i in range(6):
+    if i == 3:
+        continue
+    elif i >= 6:
```

Comment:
dont need `elif i >= 6` because the range has been defined in the for statement (line 13)